src/test/kotlin/org/jetbrains/research/transformations/util/PythonTransformationsTest.kt

```diff
@@ -0,0 +1,58 @@
+import com.github.gumtreediff.gen.python.PythonTreeGenerator
+import com.github.gumtreediff.tree.TreeContext
+import org.jetbrains.research.transformations.util.ParserSetup
+import org.jetbrains.research.transformations.util.Util
+import org.jetbrains.research.transformations.util.toXMLWithoutRoot
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.params.provider.Arguments
+import java.io.File
+import java.util.logging.Logger
+import kotlin.reflect.KFunction
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+open class PythonTransformationsTest {
+    protected val LOG = Logger.getLogger(javaClass.name)
+
+    protected val xmlTreeFileName = "${Util.getTmpPath()}/test.xml"
+
+    companion object {
+        fun getInAndOutArgumentsArray(
+            resourcesRootName: String,
+            cls: KFunction<PythonTransformationsTest>
```

Comment:
Shouldn't it be KClass instead of KFunction?

Reply:
Of course, you are right

Reply:
I am not sure how can I use KClass which inherits PythonTransformationsTest. Do you have any idea?